test/utils/image/manifest.go

```diff
@@ -71,6 +75,7 @@ var (
 	Kitten                   = ImageConfig{e2eRegistry, "kitten", "1.0", true}
 	Liveness                 = ImageConfig{e2eRegistry, "liveness", "1.0", true}
 	LogsGenerator            = ImageConfig{e2eRegistry, "logs-generator", "1.0", true}
+	MnistTpu                 = ImageConfig{e2eRegistry, "mnist-tpu", "1.0", true}
```

Comment:
Please add a comment here to clarify purpose of this image (it's easy to be considered as DevicePlugin related ...)

Reply:
Done :)